etc/kayobe/kolla.yml

```diff
@@ -285,7 +285,7 @@ kolla_build_blocks:
   # With the UCA keyring installed we can now add all repos.
   base_ubuntu_package_sources_list: |
     {% if stackhpc_kolla_clean_up_repo_mirrors | bool %}
-    COPY sources.list.ubuntu /etc/apt/sources.list.backup
+    COPY sources.list.ubuntu.noble /etc/apt/sources.list.backup
```

Comment:
https://review.opendev.org/c/openstack/kolla/+/907589 shows It's just ``sources.list.ubuntu`` for master branch. Probably because it's just noble.

Reply:
Done